packages/server/package.json

```diff
@@ -63,7 +63,10 @@
   "devDependencies": {
     "@blitzjs/core": "0.9.0",
     "@rollup/pluginutils": "^3.0.8",
-    "@types/detect-port": "^1.3.0"
+    "@types/detect-port": "^1.3.0",
+    "@types/test-listen": "^1.1.0",
+    "isomorphic-unfetch": "^3.0.0",
+    "test-listen": "^1.1.0"
```

Comment:
Can you move this up to the project root? The general rule is devDependencies go in root.